driver/opoptions/channel.go

```diff
@@ -74,3 +74,43 @@ func WithCompletePatterns(p []*regexp.Regexp) util.Option {
 		return util.ErrIgnoredOption
 	}
 }
+
+// WithInterimPromptPattern is a slice of regex patterns that are valid prompts during a send X
+// operation (either command or config as this is a channel level option). This can be used when
+// devices change their prompt to indicate adding entries to lists or something similar. For
+// example, when editing `trace-options` on nokia srl devices with scrapli(go) the prompt changes
+// to ellipses to indicate you are editing the list still, this looks like this:
```

Comment:
```suggestion
// WithInterimPromptPattern is a slice of regex patterns that are valid prompts during a send X
// operation (either command or config as this is a channel level option). This can be used when
// devices change their prompt to indicate a multiline input.
// For example, when editing `trace-options` on nokia srl devices with scrapli(go) the prompt changes
// to ellipses to indicate you are editing the list still, this looks like this:
```

Maybe rather say about multiline input? As this is a generic case, and editing a list is an implementation of a multiline input.